lighthouse-core/test/gather/gatherers/main-document-content-test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env jest */
+
+const MainDocumentContent = require('../../../gather/gatherers/main-document-content.js');
+const NetworkRecords = require('../../../computed/network-records.js');
+
+jest.mock('../../../computed/network-records.js');
+
+describe('FR compat', () => {
+  /** @type {MainDocumentContent} */
```

Comment:
a trick I like, is to just double initialize these (in declaration *and* beforeEach) to avoid duplicating the type annotation

e.g. `let gatherer = new MainDocumentGatherer();`

Reply:
(don't need to change it, just offering an alternative that comes in handy when the types get more complex :)